kotlin/add-graphics-symbols/src/main/java/com/esri/arcgisruntime/sample/addgraphicssymbols/MainActivity.kt

```diff
@@ -0,0 +1,217 @@
+          add(Point(-2.7044029178205866, 56.07153261642126))
+          add(Point(-2.698223120515766, 56.072394931722265))
+          add(Point(-2.6923866452834355, 56.07325722773041))
+          add(Point(-2.68672183108735, 56.07335303720707))
+          add(Point(-2.6812286779275096, 56.07354465544585))
+          add(Point(-2.6764221689126497, 56.074215311778964))
+          add(Point(-2.6698990495353394, 56.07488595644139))
+          add(Point(-2.6647492184479886, 56.075748196715914))
+          add(Point(-2.659427726324393, 56.076131408423215))
+          add(Point(-2.654792878345778, 56.07622721075461))
+          add(Point(-2.651359657620878, 56.076514616319784))
+          add(Point(-2.6477547758597324, 56.07708942101955))
+          add(Point(-2.6450081992798125, 56.07814320736718))
+          add(Point(-2.6432915889173625, 56.08025069360931))
+          add(Point(-2.638656740938747, 56.08044227755186))
+          add(Point(-2.636940130576297, 56.078813783674946))
+          add(Point(-2.636425147467562, 56.07728102068079))
+          add(Point(-2.637798435757522, 56.076610417698504))
+          add(Point(-2.638656740938747, 56.07507756705851))
+          add(Point(-2.641231656482422, 56.07479015077557))
+          add(Point(-2.6427766058086277, 56.075748196715914))
+          add(Point(-2.6456948434247924, 56.07546078543464))
+          add(Point(-2.647239792750997, 56.074598538729404))
+          add(Point(-2.6492997251859376, 56.072682365868616))
+          add(Point(-2.6530762679833284, 56.0718200569986))
+          add(Point(-2.655479522490758, 56.070861913404286))
+          add(Point(-2.6587410821794135, 56.07047864929729))
+          add(Point(-2.6633759301580286, 56.07028701581465))
+          add(Point(-2.666637489846684, 56.07009538137926))
+          add(Point(-2.670070710571584, 56.06990374599109))
+          add(Point(-2.6741905754414645, 56.069137194910745))
+          add(Point(-2.678310440311345, 56.06808316228391))
+          add(Point(-2.682086983108735, 56.06789151689155))
+          add(Point(-2.6868934921235956, 56.06760404701653))
+          add(Point(-2.6911850180297208, 56.06722075051504))
+          add(Point(-2.695133221863356, 56.06702910083509))
+          add(Point(-2.698223120515766, 56.066837450202335))
+          add(Point(-2.7016563412406667, 56.06645414607839))
+          add(Point(-2.7061195281830366, 56.0660708381432))
+          add(Point(-2.7100677320166717, 56.065591697864576))
+          add(Point(-2.713329291705327, 56.06520838135397))
+          add(Point(-2.7167625124302273, 56.06453756828941))
+          add(Point(-2.718307461756433, 56.06348340989081))
+          add(Point(-2.719165766937657, 56.062812566811544))
+          add(Point(-2.7198524110826376, 56.06204587471371))
+          add(Point(-2.719165766937657, 56.06166252294756))
+          add(Point(-2.718307461756433, 56.06147084563517))
+        }), SimpleLineSymbol().apply {
```

Comment:
always use constructors over the `apply` syntax if you're setting the same properties.